examples/n_monitoring.jl

```diff
@@ -61,15 +61,23 @@ for s in paths([State(2) for i in 1:N])
 end
 add_utilities!(diagram, "T", Y_T)
 
-generate_diagram!(diagram, positive_path_utility=true)
+model, z = generate_model(diagram, model_type="RJT")
 
+"""
```

Comment:
Let's start removing the bits that have been commented out

Reply:
Removed